docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -16,21 +16,20 @@ sidebar_position: 10
 
 :::tip
 
+<!-- // prettier-ignore -->
```

Comment:
why was this needed?

Reply:
Prettier forces the <Icon /> & punctuation onto new lines which renders a big dead space with that bit of copy at the bottom of the admonition. I'm sure there's a smarter way to handle these, but for now this is sort of handling artifacts from the previous site where these Font Icons are wedged into copy blocks.